### PR TITLE
Story sheriff org anzeige der alerts im livestream 178754743

### PR DESCRIPTION
**Pull request by smnscp**

### CODE REVIEW

File: src/components/DonationAlert.jsx [hunk 1]
```diff
@@ -40,7 +56,9 @@ export class DonationAlert extends React.Component {
       <br />
       {this.state.wording}
       <br />
-      {formatCents(this.props.data.donated_amount_in_cents, this.props.params) || 'Spende'} von <Nickname {...this.props.data.author} color={this.props.params.get('nicknameColor')} />
+      { this.state.data.id}
+      <br />
```
Comment (jaynetics) [inline src/components/DonationAlert.jsx:60]: is this here on purpose or just for debugging?

File: src/components/DonationAlert.jsx [hunk 2]
```diff
@@ -24,10 +29,21 @@ export class DonationAlert extends React.Component {
...
+    if (this.props.data.id && this.props.data.id !== this.state.data.id
+      // At least one data field must be present.
+      && (this.props.data.donated_amount_in_cents || this.props.data.author)) {
+      console.debug("componentDidUpdate YES", prevProps)
+      this.setState({
+        hidden: false,
+        data: {
+          id: this.props.data.id,
+          author: this.props.data.author,
+          amountInCents: this.props.donated_amount_in_cents,
```
Comment (jaynetics) [inline src/components/DonationAlert.jsx:42]:
```suggestion
          amountInCents: this.props.data.donated_amount_in_cents,
```

i guess?

**Review by jaynetics — COMMENTED** [review]
two questions in the comments, looks good otherwise!

File: src/components/DonationAlert.jsx (hunk 1 above)
Comment (smnscp) [reply src/components/DonationAlert.jsx:60]: Must have been Friday. 🤦

**Pull request merged**